graphql/src/test/java/com/scalar/db/graphql/GraphQlFactoryTest.java

```diff
@@ -0,0 +1,224 @@
+package com.scalar.db.graphql;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import com.scalar.db.api.DistributedStorage;
+import com.scalar.db.api.DistributedStorageAdmin;
+import com.scalar.db.api.DistributedTransactionManager;
+import com.scalar.db.api.TableMetadata;
+import com.scalar.db.exception.storage.ExecutionException;
+import com.scalar.db.graphql.schema.CommonSchema;
+import com.scalar.db.graphql.schema.Constants;
+import com.scalar.db.io.DataType;
+import com.scalar.db.service.StorageFactory;
+import com.scalar.db.service.TransactionFactory;
+import graphql.GraphQL;
+import graphql.schema.GraphQLNamedInputType;
+import graphql.schema.GraphQLObjectType;
+import graphql.schema.GraphQLSchema;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+public class GraphQlFactoryTest {
+  private static final String NAMESPACE_NAME = "namespace_1";
+  private static final String TABLE_NAME_1 = "table_1";
+  private static final String TABLE_NAME_2 = "table_2";
+  private static final String COLUMN_NAME_1 = "column_1";
+  private static final String COLUMN_NAME_2 = "column_2";
+  private static final String COLUMN_NAME_3 = "column_3";
+
+  @Mock private StorageFactory storageFactory;
+  @Mock private TransactionFactory transactionFactory;
+  @Mock private DistributedStorage storage;
+  @Mock private DistributedStorageAdmin storageAdmin;
+  @Mock private DistributedTransactionManager transactionManager;
+
+  @Before
+  public void setUp() {
+    initMocks(this);
```

Comment:
We introduced Mockito 3 in #348, and `MockitoAnnotations.initMock()` is deprecated and replaced with `MockitoAnnotations.openMocks()` in Mockito 3. Can you please use `MockitoAnnotations.openMocks()`?